rwengine/src/engine/GameWorld.cpp

```diff
@@ -788,20 +784,32 @@ void GameWorld::startCutscene() {
 }
 
 void GameWorld::clearCutscene() {
+    eraseCutsceneObjects();
+    eraseCutsceneSound();
+    eraseCutsceneAnimations();
+
+    state->currentCutscene = std::nullopt;
+    state->isCinematic = false;
```

Comment:
maybe we should prepare "universal" function to free camera?

Reply:
If it would mean, less spaghetti code, fine by me :smile: